consoleApp/src/main/java/com/infoshare/UserRegistration.java

```diff
@@ -0,0 +1,133 @@
+package com.infoshare;
+
+import com.infoshare.domain.TypeOfHelp;
+import com.infoshare.domain.Volunteer;
+
+import java.util.Scanner;
+
+public class UserRegistration {
+    private static final String REGISTRATION_HEADER = "[Witaj w programie do rejestracji]";
+    private static final String REGISTRATION_NAME = "Podaj imie/nickname (min 3 znaki): ";
+    private static final String REGISTRATION_LOCATION = "Podaj swoją lokalizację (bez polskich znaków): ";
+    private static final String REGISTRATION_PHONE_NUMBER = "Podaj numer telefonu: ";
+    private static final String REGISTRATION_EMAIL = "Podaj e-mail";
+    private static final String REGISTRATION_HELP_TYPE = "W czym możesz pomóc innym, wybierz z listy: ";
+    private static final String REGISTRATION_ERROR = "***BŁĄD***";
+
+    private String getVolunteerName(){
+        String username = "";
+        boolean isGood = false;
+        do{
+            System.out.println(REGISTRATION_NAME);
+            Scanner sc = new Scanner(System.in);
+            username = sc.nextLine();
+            if(username.length() > 2){
+                isGood = true;
+            }else{
+                System.out.println(REGISTRATION_ERROR);
+            }
+        }while (isGood == false);
+
+        return username;
+    }
+
+    private String getLocation(){
+        String location = "";
+        boolean isGood = false;
+        do{
+            System.out.println(REGISTRATION_LOCATION);
+            Scanner sc = new Scanner(System.in);
+            location = sc.nextLine();
+            if(isAlpha(location) == true){
+                isGood = true;
+            }else {
+                System.out.println(REGISTRATION_ERROR);
+            }
+        }while (isGood == false);
+        return location;
+    }
+
+    private String getPhoneNumber(){
+        String phone = "";
+        boolean isGood = false;
+        do{
+            System.out.println(REGISTRATION_PHONE_NUMBER);
+            Scanner sc = new Scanner(System.in);
+            phone = sc.nextLine();
+            if(isNumber(phone) == true){
+                isGood = true;
+            }else {
+                System.out.println(REGISTRATION_ERROR);
+            }
+        }while (isGood == false);
+        return phone;
+
+    }
+
+    private String getEmail(){
+        String email = "";
+        boolean isGood = false;
+        do{
+            System.out.println(REGISTRATION_EMAIL);
+            Scanner sc = new Scanner(System.in);
+            email = sc.nextLine();
+            if(isEmailValid(email) == true ){
+                isGood = true;
+            }else {
+                System.out.println(REGISTRATION_ERROR);
+            }
+        }while (isGood == false);
+        return email;
+    }
+
+    private String getHelpType(){
```

Comment:
Przydałby się blok try/catch. Wpisujemy tutaj int, więc jeśli ktoś wpisze np. _2 to wywali InputMismatchException i zamknie program

Reply:
zrobione :+1: